runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -7416,6 +7434,11 @@ done:;
 						goto done;
 					}
 				}
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
```

Comment:
Unnecessary if the out of line allocator does the right thing (and we fail out to it correctly!).